paper/main.tex

```diff
@@ -415,22 +414,22 @@ \subsubsection*{Recovering dynamic higher-order correlations}
 \subsection*{Cognitively relevant dynamic high-order correlations in
   fMRI data}
 We used across-participant temporal decoders to identify cognitively
-relevant neural patterns in fMRI data (see \textit{Forward inference
-  and decoding accuracy}).  The dataset we examined~\citep[collected
+relevant neural patterns in fMRI data (see Forward inference
+  and decoding accuracy).  The dataset we examined~\citep[collected
 by][]{SimoEtal16} comprised four experimental conditions that exposed
 participants to stimuli that varied systematically in how cognitively
-engaging they were.  The \textit{intact} experimental condition had
+engaging they were.  The intact experimental condition had
 participants listen to an audio recording of a 10-minute story.  The
-\textit{paragraph}-scrambled experimental condition had participants
+paragraph-scrambled experimental condition had participants
 listen to a temporally scrambled version of the story, where the
 paragraphs occurred out of order (but where the same total set of
 paragraphs were presented over the full listening interval).  All
 participants in this condition experienced the scrambled paragraphs in
-the same order.  The \textit{word}-scrambled experimental condition
+the same order.  The word-scrambled experimental condition
```

Comment:
change to:

the same order.  The word-scrambled experimental condition (word)...

Reply:
done